### ISSUE TITLE
Documentation Incorrect: No Config Created

### ISSUE THREAD

**Issue by MrHinsh** [microsoft/process-migrator#84]
> Just run process-migrator without any argument will create the file if it does not exist.

This is not the case.. it just errors out.

**Comment by MOlausson** [comment]
Hey @MrHinsh, can you share a bit more context. For me it does create the template file just fine (tested on Codespace and Windows).

**Comment by MrHinsh** [comment]
If I install the tool and then just run it from the command prompt with no argument I get:

>>[ERROR] [2023-08-15T09:21:22.011Z] [Configuration validation] Missing or invalid source account url: 'Required in 'export'/'migrate' mode, source account url.'.

![image](https://github.com/microsoft/process-migrator/assets/5205575/7d6effa2-287f-47a1-99e0-34668520fdfc)

**Comment by MrHinsh** [comment]
OK, so there was an old configuration.json in that folder. 

Perhaps if it finds a config it should say "Loading config: C:\Users\MartinHinshelwood(He\configuration.json" so that it is more obvious that it is indeed finding a config, but that its format is not valid.

**Comment by MOlausson** [comment]
I agree that the documentation (and error handling) can be improved. The team will be maintaining the tool more actively going forward so will make sure it get prioritized.
